### PR TITLE
Create troubleshooting.md

### PR DESCRIPTION
**Pull request by benedict-notbot**
Add methods for self-service restarting SSL validation

### CODE REVIEW

**Comment by kodster28** [comment]
Howdy @benedict-notbot, since I made a few edits I'm reaching out to my colleagues for a review.

File: products/ssl/src/content/edge-certificates/universal-ssl/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+order: 5
+pcx-content-type: faq
+---
+
+# Troubleshooting
+
+## Resolving a timed out state
```
Comment (dcpena) [inline products/ssl/src/content/edge-certificates/universal-ssl/troubleshooting.md]: Nit, but I'd change the verb to be Resolve

File: products/ssl/src/content/edge-certificates/universal-ssl/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+If a certificate issuance times out, Cloudflare tell you where in the chain of issuance the timeout occurred: Initializing, Validation, Issuance, Deployment, or Deletion.
```
Comment (dcpena) [inline products/ssl/src/content/edge-certificates/universal-ssl/troubleshooting.md]:
```suggestion
If a certificate issuance times out, Cloudflare tells you where in the chain of issuance the timeout occurred: Initializing, Validation, Issuance, Deployment, or Deletion.
```

File: products/ssl/src/content/edge-certificates/universal-ssl/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+To resolve timeout issues:
+
+* Change the **Proxy status** of related DNS records to **DNS only** (gray-clouded) and wait at least a minute. Then, change the **Proxy status** back to **Proxied** (orange-clouded).
```
Comment (dcpena) [inline products/ssl/src/content/edge-certificates/universal-ssl/troubleshooting.md:14]: Do these items needed to be completed in this order? If yes, consider swapping out for a numbered list.

**Review by dcpena — APPROVED** [review]
Approved with small comments

File: products/ssl/src/content/edge-certificates/universal-ssl/troubleshooting.md (hunk 3 above)
Comment (kodster28) [reply products/ssl/src/content/edge-certificates/universal-ssl/troubleshooting.md:14]: Nope! I added a bit of verbiage at the front of the list to help make that clearer.

**Pull request merged**